pkg/cli/cmds/agent.go

```diff
@@ -151,6 +153,18 @@ var (
 		Usage:       "(agent/networking) Override default flannel cni config file",
 		Destination: &AgentConfig.FlannelCniConfFile,
 	}
+	VPNAuth = &cli.StringFlag{
+		Name:        "vpn-auth",
+		Usage:       "(agent/networking) Credentials for the VPN provider. It must include the provider name and join key in the format name=foo,joinKey=var",
```

Comment:
Maybe using valid value could it be more understandable `name=tailscale,joinKey=<key>`

Reply:
yeah, I can do that. `name=tailscale` might be good right now but if we add more vpns, then it may confuse people. Something to consider in the future

Reply:
even just `name=PROVIDER,joinKey=MYKEY` might be more clear?

Reply:
Maybe if we decided to mark this as experimental could be better to write it on the flag description?

Reply:
Done! :)

Reply:
I think that also `vpn-auth-file` should be marked as experimental

Reply:
yeah, good catch